src/components/data_comps/desp/cime_config/namelist_definition_desp.xml

```diff
@@ -57,7 +57,7 @@
       Level of debug output, 0=minimum, 1=normal, 2=more, 3=too much (default: 0)
     </desc>
     <values>
-      <value>0</value>
+      <value>1</value>
```

Comment:
Okay, I just took out info_debug so you should see all the messages all the time (well, one warning only shows up if you are using DATATEST mode, not in NOCHANGE or NULL mode).